resources/configure-metrics/cwagent-deployment.yaml

```diff
@@ -0,0 +1,150 @@
+---
+# create configmap for prometheus cwagent config
+kind: ConfigMap
+metadata:
+  name: prometheus-cwagentconfig
+  namespace: amazon-cloudwatch
+apiVersion: v1
+data:
+  # cwagent json config
+  cwagentconfig.json: |
+    {
+      "agent": {
+        "region": "__cluster_region__",
+        "debug": true
+      },
+      "logs": {
+        "metrics_collected": {
+          "prometheus": {
+            "cluster_name": "__cluster_name__",
+            "log_group_name": "/aws/containerinsights/__cluster_name__/prometheus",
```

Comment:
Wrong placeholders: we use `<CHANGE_ME>` 
To fix everywhere

Reply:
Done